index.js

```diff
@@ -4,7 +4,7 @@ log.time("startup");
 log.info("Starting Cutiebot!");
 
 const Discord = require("discord.js");
-const { token } = require("./config.json");
+const token = process.env.DISCORD_TOKEN;
```

Comment:
I would add a fast failure state when token missing, like "Token not provided please set `DISCORD_TOKEN` env variable" or something